src/cli/java/org/commcare/util/screen/MultiSelectEntityScreen.java

```diff
@@ -16,7 +16,7 @@
  */
 public class MultiSelectEntityScreen extends EntityScreen {
 
-    private static final CharSequence USE_SELECTED_VALUES = "use_selected_values";
+    public static final CharSequence USE_SELECTED_VALUES = "use_selected_values";
```

Comment:
Why does this need to be public? Looks like it's only referenced within this class.